pkg/client/dtclient/client.go

```diff
@@ -470,6 +470,12 @@ func (d *DynatraceClient) ConfigExistsByName(ctx context.Context, api api.API, n
 }
 
 func (d *DynatraceClient) configExistsByName(ctx context.Context, api api.API, name string) (exists bool, id string, err error) {
+	if api.SingleConfiguration {
+		// check that a single configuration is there by actually reading it.
+		_, err := d.readConfigById(ctx, api, "")
+		return err == nil, "", nil
```

Comment:
Probably better error handling is needed here

Reply:
should we move that check to https://github.com/Dynatrace/dynatrace-configuration-as-code/blob/cf3c3e5fba87421a4207ceb20df3401684b8d361/pkg/client/dtclient/config_client.go#L468-L469 ?

Reply:
conceptually, do SingleConfigurationObjects have IDs? I was thinking the scoped ones have a scope, but not really an ID of their own